lib/screen/book_editor.dart

```diff
@@ -120,18 +127,42 @@ class _BookEditorState extends State<BookEditor> {
   }
 
   Widget showBookCover() {
```

Comment:
I think function name should be "`getBookCover`" instead?

Reply:
I named it like that because it is just a widget and it shows the book when it is available by checking the cover attribute